corehq/apps/domain/models.py

```diff
@@ -323,7 +323,8 @@ class Domain(QuickCachedDocumentMixin, Document, SnapshotMixin):
     sms_survey_date_format = StringProperty()
 
     # Allowed outbound SMS per day
-    daily_outbound_sms_limit = IntegerProperty(default=5000)
+    # If this is None, then the default is applied. See get_daily_outbound_sms_limit()
+    custom_daily_outbound_sms_limit = IntegerProperty()
```

Comment:
should this have `null=True` so it allows null values? Might need a migration to remove the `NOT NULL` constraint if one exists. Ignore this comment if `null` defaults to `True` (although I think [it does not](https://docs.djangoproject.com/en/2.0/ref/models/fields/#null)).

Reply:
this is actually on a couch model so it will allow null values, and also it will default to null if no value is given when saving or if it's missing from the json (good point though, i had tested that behavior to make sure it's what we want)

Reply:
ha, couch! my bad